### PR TITLE
Update documentation (README and package.json)

### PR DESCRIPTION
**Pull request by rnchamberlain**
Add version and platform support statements, make setEvents()
API more obvious, provide REPL example. For package.json correct
main, add bugs link, improve homepage link.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,8 +14,12 @@
   "dependencies": {
     "nan": "^2.3.5"
   },
+  "author": "Richard Chamberlain",
```
Comment (cjihrig) [inline package.json]: Since this is now a foundation project, maybe this should be `contributors` like `nan` does.

Comment (rnchamberlain) [inline package.json]: Ah yes, array of` "contributors": [ ` - thanks, will change

**Comment by sam-github** [comment]
Would be better to separate into two commits, and have them describe the changes (like `package: set metadata (author, homepage, version, ...)`)

**Comment by rnchamberlain** [comment]
OK, I will raise a new PR with two separate commits.

**Pull request closed without merging**